frontend/src/component/menu/Header/Header.tsx

```diff
@@ -54,9 +54,7 @@ const StyledContainer = styled(Container)(({ theme }) => ({
     display: 'flex',
     alignItems: 'center',
     maxWidth: 1280,
-    [theme.breakpoints.down('md')]: {
-        padding: '0',
-    },
+    '&&&': { padding: 0 },
```

Comment:
i used this method because there is some styling that is applied to these component and overwrite a normal padding